app/styles/config.js

```diff
@@ -0,0 +1,24 @@
+import ENV from '../config/environment';
```

Comment:
Oops, I'm guessing you didn't mean to put the mirage config into the style's directory